azure-pipelines.yml

```diff
@@ -179,9 +179,50 @@ stages:
                testRunTitle: Functional Test Results
              displayName: 'Publish Functional Test Results'
 
-      - job: AccessibilityTests
+      - job: E2ETests
         dependsOn: FunctionalTests
```

Comment:
Accessibility, FT's and E2E all can run in parallel unless we have some dependencies. Can we remove depends on?

Reply:
Removed DependsOn from ATs , ETs